x-pack/elastic-agent/Jenkinsfile.yml

```diff
@@ -85,3 +85,7 @@ stages:
     #            - "windows-7-32"
     #        branches: true     ## for all the branches
     #        tags: true         ## for all the tags
+    packaging-linux:
+        packaging-linux: "mage package"
+        e2e:
+            enabled: false
```

Comment:
is this where we'd set it to true to run the e2e-testings for Agent?  @mdelapenya @v1v not sure what else we need to do here, but along with all the Beats, Agent urgently needs this to help keep tests passing and quality quantified.

Reply:
@EricDavisX , this is an improvement for running the packaging with e2e within the main pipeline. So far, I'm working on iterations, so testing one of the beats at the time. 

This improvement and the existing one are complementaries, and I won't change the existing workflow, until we feel safe to do so.

The existing workflow runs as below:

* on branches, it runs the main pipeline (build/lint/testing), then the packaging pipeline and finally the e2e pipeline.
* on pr basis, it runs the main pipeline, but the packaging and e2e pipelines are not triggered by default, it requires a GitHub comment to enable it.

This proposal want to improve the automation on a PR basis and skip the GitHub comments, since not every contributor uses it and worth to run the e2e on a PR basis to catch errors before any merge.

Please let me know if I need to clarify this further